hr_employee_exemption/hr_employee_exemption.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding:utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Savoir-faire Linux. All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import orm, fields
+
+
+class hr_employee_exemption(orm.Model):
+    _name = 'hr.employee.exemption'
+    _description = 'Employee Income Tax Exemption'
+
+    _columns = {
+        'employee_id': fields.many2one(
+            'hr.employee',
+            'Employee',
+            required=True,
+            ondelete='cascade',
+        ),
+        'exemption_id': fields.many2one(
```

Comment:
Since it is quite directly linked with the payslip, and from my point of view it affects a salary rule (i don't know how you structure the salaries, but for me, in salary rule you have gross, benefits, deductions, employee taxes, employer taxes, and net), isn't it easier to directly link the exemption_id with a salary rule, and on compute payslip the amount to be 0...what do you think??

@pedrobaeza @bwrsandman @max3903